src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month month--length-28">
+      <div
+        class="month__item month__item--start-day-fri"
+      ></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+      <div class="month__item"></div>
+    </div>
```

Comment:
```suggestion
    <div class="calendar calendar--month-length-28 calendar--start-day-fri">
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
    </div>
```

Reply:
![image](https://github.com/mate-academy/layout_calendar/assets/55200530/7f7427ad-244f-4da0-abf9-4c0b61f59e39)

it should be `calendar` block